server/server.ts

```diff
@@ -31,6 +34,34 @@ app.get('/', (_req: Request, res: Response) => {
     .sendFile(path.resolve(import.meta.dirname, '../client/index.html'));
 });
 
+app.post(
+  '/api',
+  // userQueryController,
+  openaiNLPQuery,
```

Comment:
@nanyi-deng I think the reason you are getting linting errors here is because you need to merge with dev